roles/tackle/templates/deployment-ui.yml.j2

```diff
@@ -37,6 +37,32 @@ spec:
         role: {{ ui_service_name }}
     spec:
       containers:
+{% if feature_auth_required|bool %}
+{% if feature_auth_type == "openshift-oauth" %}
+        - args:
+          - --https-address=:8443
+          - --provider=openshift
+          - --tls-cert=/etc/tls/private/tls.crt
+          - --tls-key=/etc/tls/private/tls.key
+          - --upstream=http://localhost:8080
+          - --openshift-service-account={{ ui_serviceaccount_name }}
+          - --openshift-sar={"namespace":"{{ app_namespace }}","resource":"services","resourceName":"{{ ui_service_name }}","verb":"get"}
```

Comment:
I wonder if there is a better approximation? but honestly don't know

Reply:
Is this in reference to the openshift-sar rule? I took this from the example at https://github.com/openshift/oauth-proxy#openshift-oauth-proxy

If there's something better I am happy to change it.

Reply:
At the least we can probably make this or an alternative the value for a default `oauth_openshift_sar`. If someone then wants to get inventive and change the sar rule for their environment they can do so.

Reply:
that makes sense, I was just thinking about what it would be that the users care to give access to to allow someone to access the UI. I imagine that this will be fine for the default.